src/zx/FunctionalityConstruction.cpp

```diff
@@ -72,30 +79,28 @@ namespace zx {
 
         const auto t0 = diag.addVertex(target, diag.getVData(qubits[target]).value().col + 1);
         const auto t1 = diag.addVertex(ctrl, diag.getVData(qubits[target]).value().col + 1);
-
         diag.addEdge(s0, t1);
         diag.addEdge(s1, t0);
-
         qubits[target] = t0;
         qubits[ctrl]   = t1;
     }
 
     void FunctionalityConstruction::addCcx(ZXDiagram& diag, Qubit ctrl0, Qubit ctrl1, Qubit target,
```

Comment:
```suggestion
    void FunctionalityConstruction::addCcx(ZXDiagram& diag, const Qubit ctrl0, const Qubit ctrl1, const Qubit target,
```

Reply:
Actually, these have already been unconsted in previous versions. So we had a whole release with unconsted variables.

Reply:
😱 how could that have happened